templates/deployment.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
       annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
This enforces a new deploy on each new release. Until now there wasn't any clear way to reload the varnish pods when there was an VCL change.

I think it's a good tradeoff.

Reply:
Yes, I've also spent some time to try understand how we could deploy the config to achieve this, but this is definitely better than having to roll the deployment manually on config change.